repo/packages/D/data-science-engine/0/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.12",
+  "name": "data-science-engine",
+  "version": "1.0.0",
+  "scm": "https://github.com/mesosphere/jupyter-service",
+  "maintainer": "support+data-science-engine@mesosphere.com",
+  "website": "https://docs.mesosphere.com/services/data-science-engine",
+  "description": "DC/OS Data Science Engine provisions Jupyter Notebooks, an open-source web application that allows you to create and share documents that contain live code, equations, visualizations and narrative text.\nUses include: data cleaning and transformation, numerical simulation, statistical modeling, data visualization, machine learning, and much more.",
+  "framework": true,
+  "selected": true,
+  "upgradesFrom": [],
+  "downgradesTo": [],
+  "tags": [
+    "beakerx",
+    "dask",
+    "hdfs",
+    "jupyter",
+    "jupyterlab",
+    "notebook",
+    "openid-connect",
+    "ray",
+    "s3",
+    "spark",
+    "tensorflow"
+  ],
+  "preInstallNotes": "Default configuration requires 1 agent node with: 1 CPU | 4096 MB MEM",
```

Comment:
Should we add a note about HDFS requiring an additional 5 nodes?

Reply:
I don’t think it’s relevant because this requirement is stated in HDFS pre-install notes